### DISCUSSION TITLE
[Dataset/Title]: CEMS Fire Danger Indices

### DISCUSSION THREAD

**Discussion by andrewclelland** [samapriya/awesome-gee-community-datasets#302] [General]
### Contact Details

andell84@bas.ac.uk

### Provide Dataset Link

https://gee-community-catalog.org/projects/cems_fire/

### Describe the update

When working with the CEMS FWI dataset loaded from the GEE Community Catalog I receive the following exception: `ee.ee_exception.EEException: Failed to read object "gs://ce-cems-fire-daily-4-1/20070405.tif#1692071882764". Invalid authentication from policy (go/gcs-rpc-sp): gaiamint expired: mint expired, now=2024-10-00:44:43.146402Z, notAfter=2024-10-06T00:44:43Z, origin_identifier=gestalt, gaiamint_server=backend::1736c6da1cb9f-4db81ca8ab1.borgtask.google.com:9801; RpcSecurityPolicy http://rpcsp/p/HaoQuKpjPQy0AsUvaqGWq5y_lNBpDEUHQZfcf3AQ`. Is this something you can fix on your end? Thanks

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samapriya** [comment]
I Andrew I am going to reach out to the maintainers of this dataset for this one . Converting this to a discussion for now

**Comment by andrewclelland** [comment]
Many thanks! I have also just found one other file where this error occurs:
`Error: Failed to read object "gs://ce-cems-fire-daily-4-1/20231205.tif#1709424684491404". No such object: ce-cems-fire-daily-4-1/20231205.tif (Error code: 3)`

**Comment by samapriya** [comment]
This has now been fixed as per the authors, please let me know if you are still seeing errors. Closing for now feel free to reopen the discussion
